src/js/themes/base.js

```diff
@@ -379,6 +379,11 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       },
       // color: { dark: undefined, light: undefined }
       primary: {
+        active: {
+          border: {
+            // color: { dark: undefined, light: undefined }
```

Comment:
According to the documentation you added, this value should default to `control` color, according to this impl it should default to undefined.

Reply:
Gotcha -- so right now, to maintain backwards compatibility, the border falls back to 'control' color unless overridden by the theme. Should I changes the docs to say 'undefined', or should I update the default color to 'control'?

Reply:
I think it will be better to set it to `control` on the primary.active.border theme, and remove the default fallback on the code, otherwise, the color wouldn't be documented anywhere since it is a code side-effect and not part of the theme or props default behavior.

Reply:
Sounds good. I like that approach.